src/main/java/teammates/ui/controller/InstructorCourseJoinConfirmationPageData.java

```diff
@@ -1,14 +1,43 @@
 package teammates.ui.controller;
 
 import teammates.common.datatransfer.AccountAttributes;
+import teammates.common.util.Const;
+import teammates.common.util.Sanitizer;
 
 public class InstructorCourseJoinConfirmationPageData extends PageData {
 
-    public String regkey;
-    public String institute;
+    private String regkey;
+    private String institute;
     
     public InstructorCourseJoinConfirmationPageData(AccountAttributes account) {
         super(account);
     }
+    
+    public void init(String regkey, String institute) {
+        this.regkey = regkey;
+        this.institute = institute;
+    }
```

Comment:
We can probably combine the constructor with `init()` here without any loss in information.